### ISSUE TITLE
Expand the range of comments recognised.

### ISSUE THREAD

**Issue by SundareshSankaran** [sassoftware/sas-copilot#1]
**Is your feature request related to a problem? Please describe.**

SAS has multiple ways of inserting comments in code.  Even Copilot provided the following as suggestions:

```sas
/* This is a traditional style comment in SAS */
/******************************************************
* This is a block-style comment in SAS
* that spans multiple lines.
******************************************************/
* This is an alternative single-line comment in SAS ;
/********************************************************************************************************
/* This is a nested comment in SAS within another comment.
   It can be useful for adding additional details to a comment block. */
********************************************************************************************************/
```

And @cjdinger  also suggested this, which was new to me (perhaps from old versions of SAS):

```sas
COMMENT: this is a comment;
```

Currently, the "Generate code" prompt suggestion appears only for the following style:
```sas
/* This is a traditional style comment in SAS */
```

It doesn't appear for the following alternatives, at least (additional testing underway)

```sas
COMMENT: this is a comment;

* This is an alternative single-line comment in SAS ;

```

**Describe the solution you'd like**

It would be nice to have more comment "styles" get recognised as comments and treated accordingly.

Consider:  As a consequence of the SAS 'legacy', there are many programmers with technical debt and entrenched styles and we might like to consider their preferences.  

**Describe alternatives you've considered**

This isn't a showstopper, to be honest.  One can **still** right-click and select Generate code.  It just doesn't come up for all type of comment styles.  Hopefully they can be covered.

**Additional context**

Providing some screenshots showing what happens, and what works current, and what's desired.

<img width="846" alt="want_2" src="https://github.com/sassoftware/sas-copilot/assets/37813227/5fba9125-be47-46b5-85de-1c19c565768a">
<img width="846" alt="works" src="https://github.com/sassoftware/sas-copilot/assets/37813227/b4e22994-1755-4a81-b0a7-410f760af422">
<img width="846" alt="want" src="https://github.com/sassoftware/sas-copilot/assets/37813227/11c456b7-4414-44d5-a56e-f086b1cbf891">

**Reporter's Organization**

SAS- GTP

**Comment by Criptic** [comment]
Adding the macro comment facility for future macro enhancements here as well
%* my comment;

**Comment by ouhanyou** [comment]
Hi @cjdinger @SundareshSankaran, I tried out the `COMMENT: this is a comment;` in SAS studio but it doesn't seem to work. Did I mis-type anything?
<img width="1726" alt="image" src="https://github.com/sassoftware/sas-copilot/assets/105657176/232a93a8-ed7d-4ade-9f7b-43d01e7df03c">

**Comment by ouhanyou** [comment]
Thanks for opening this issue @SundareshSankaran !
I'm working on supporting these three comment methods.
1. `/* This is a traditional style comment in SAS */` (already implemented)
2. `%* this is a comment;`
3. `* this is a comment;`
It should be available soon.

**Comment by ouhanyou** [comment]
This issue has been fixed in version 0.0.4, so I'm closing this one. Thanks!
